package/kernel/qca-ssdk/patches/0003-qca8081-add-LED-rule-for-Arcadyan-AW1000.patch

```diff
@@ -0,0 +1,90 @@
+From 0bbcb1f0f5dc398c52f3adc4d671cb7b3b61bf2a Mon Sep 17 00:00:00 2001
+From: Chukun Pan <amadeus@jmu.edu.cn>
+Date: Wed, 18 Aug 2023 23:06:01 +0800
+Subject: [PATCH] qca8081: add LED rule for Arcadyan AW1000
+
+Some routers, like Arcadyan AW1000, the LED of qca8081 is
+connected to the LED2 register and active low.
+
+Add a check for system compatible as we can parse it
+from device tree to make it work.
+
+Signed-off-by: Chukun Pan <amadeus@jmu.edu.cn>
+---
+ include/hsl/phy/qca808x_phy.h |  6 +++---
+ src/hsl/phy/qca808x_phy.c     | 30 +++++++++++++++++++++++++++---
+ 2 files changed, 30 insertions(+), 6 deletions(-)
+
+--- a/include/hsl/phy/qca808x_phy.h
++++ b/include/hsl/phy/qca808x_phy.h
+@@ -428,9 +428,9 @@ extern "C"
+ #define QCA808X_PHY_MMD7_LED1_CTRL              0x8074
+ #define QCA808X_PHY_MMD7_LED2_CTRL              0x8076
+ #define QCA808X_PHY_MMD7_LED_POLARITY_ACTIVE_HIGH 0x46
+-#define QCA808X_PHY_MMD7_LED0_CTRL_ENABLE       0x8670
+-#define QCA808X_PHY_MMD7_LED1_CTRL_DISABLE      0x0
+-#define QCA808X_PHY_MMD7_LED2_CTRL_DISABLE      0x0
++#define QCA808X_PHY_MMD7_LED_POLARITY_ACTIVE_LOW  0x6
```

Comment:
Why are these being renamed?

Reply:
It looks like the CTRL_ENABLE and CTRL_DISABLE values are the same for LED 1/2/3.

Reply:
This patch wont apply anymore as QCA changed the API so they now have MMD modify directly

Reply:
Update, although it seems a bit redundant, is it possible now?

Reply:
We are going to drop SSDK QCA8081 driver anyway as @Ansuel has added LED support to the upstream driver.

So, it would just add code that is going to get dropped rather soon.